content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Feature onboarding
+---
+
+import {Box, Flex, BorderBox, Text} from '@primer/components'
+
+This is a guide for designing onboarding for the product and does not include what to do for marketing pages, email announcements, social media, etc.
+
+Onboarding is a virtual unboxing experience that helps users get started with an app.
+
+- ✅ Think creatively utilizing different ways to onboard users, such as empty states and off-product avenues.
+- ✅ For feature annoucements, prepare a release timeline to call out new features.
+- 🚫 Do not leave feature annoucement campaigns running forever.
+
+# Guiding principles
+## Proximity and proportion
+Onboarding elements should maintain an appropriate proximity to where the feature will live in perpetuity.
+
+Consider the primary job to be done on a page before disrupting the user with a feature callout.  Keep in mind the context, placement, and dominance of an onboarding element. Onboarding UI choices should be proportional to the hierarchy of the feature on the page.
+
+## Set clear campaign limits
+Use feature onboarding UI for a limited time, make it clear to the user how they may dismiss the message and return to their original task quickly. Define clear paramaters for why, when, and how often a user may see a onboarding message:
```

Comment:
UI --> components?

Reply:
These above (3 sentences) could also be a list. I'm just not sure who to incorporate the list below without having 2 indentation levels, but maybe that's fine? Or it's a separate list.